dashboard/views.py

```diff
@@ -467,7 +467,7 @@ def grade_distribution(request, course_id=0):
             'course_id': course_id,
             'enrollment_type': 'StudentEnrollment'
         })
-    if df.empty or df.count().current_grade < 6:
+    if df.empty or df.count().current_grade < 6 or config.GRADE_DISTRIBUTION_LIMIT > len(df):
```

Comment:
I think we should have a different warning here if it gets blocked by the limit? What do you think?

Like if he grades are less than 6 display this error message.

But if the `config.GRADE_DISTRIBUTION_LIMIT > len(df)` say something like
Course enrollments for this course are less than the configured minimum of {config.GRADE_DISTRIBUTION_LIMIT}.

Also is this logic correct? Are we blocking it if the LIMIT is greater than the length? The default limit (which describes a minimum) is set to 10. So this seem like it would return this error message if there was 11?

I feel like this should be reversed.

Also maybe the name of `GRADE_DISTRIBUTION_LIMIT` should be `GRADE_DISTRIBUTION_FLOOR` or `GRADE_DISTRIBUTION_MINiMUM` as it seems like this is a lower limit?

Reply:
I made changes based on your comment, Please review